src/github.com/getlantern/golog/golog_test.go

```diff
@@ -112,3 +113,24 @@ func TestAsStdLogger(t *testing.T) {
 	stdlog.Printf("Hello %d", 5)
 	assert.Regexp(t, severitize("ERROR", expectedStdLog), string(out.Bytes()))
 }
+
+func newBuffer() *synchronizedbuffer {
+	return &synchronizedbuffer{orig: &bytes.Buffer{}}
+}
+
+type synchronizedbuffer struct {
+	orig  *bytes.Buffer
+	mutex sync.RWMutex
```

Comment:
minor nit, but this seems like a good place to use an anonymous member of the struct, i.e. `sync.RWMutex` versus `mutex sync.RWMutex` and `buf.Lock()` versus `buf.mutex.Lock()`